frontend/src/app/login/page.tsx

```diff
@@ -56,48 +56,85 @@ function LoginInner() {
-              setError("");
-              setPassword(e.target.value);
-            }}
-            className="bg-zinc-200 p-2 w-full"
-            placeholder="password"
-          />
-          <button
-            className={`bg-black ${
-              loggingIn ? "bg-gray-600" : "bg-black"
-            } text-white p-2 w-2/5`}
-            onClick={handleLogin}
-          >
-            {!loggingIn ? "Login" : "Logging in..."}
-          </button>
+        <div className="sm:mx-auto sm:w-full sm:max-w-sm">
+          <h2 className="mt-10 text-center text-2xl font-bold leading-9 tracking-tight text-gray-900">
+            Sign in to your Hairify account
+          </h2>
         </div>
-        <div className="text-sm">
-          {`Don't have an account ?`}{" "}
-          <span className="text-blue-700 cursor-pointer">
-            <Link href="signup">Signup</Link>
-          </span>
+
+        <div className="mt-10 sm:mx-auto sm:w-full sm:max-w-sm">
+          <form className="space-y-6" action="#" method="POST">
+            <div>
+              <label htmlFor="email" className="block text-sm font-medium leading-6 text-gray-900">
+                Email address
+              </label>
+              <div className="mt-2">
+                <input
+                  id="email"
+                  name="email"
+                  type="email"
+                  autoComplete="email"
+                  placeholder="email"
+                  value={email}
+                  onChange={(e) => {
+                    setError("");
+                    setEmail(e.target.value);
+                  }}
+                  required
+                  className="block px-3 w-full rounded-md border-0 py-1.5 text-gray-900 placeholder:text-gray-400 text-sm sm:leading-6"
+                  style={{ outline: 'none' }}
+                />
+                  </div>
+            </div>
+
+            <div>
+              <div className="flex items-center justify-between">
+                <label htmlFor="password" className="block text-sm font-medium leading-6 text-gray-900">
+                  Password
+                </label>
+              </div>
+              <div className="mt-2">
+                <input
+                  id="password"
+                  name="password"
+                  type="password"
+                  autoComplete="current-password"
+                  required
+                  placeholder="password"
+                  value={password}
+                  onChange={(e) => {
+                    setError("");
+                    setPassword(e.target.value);
+                  }}
+                  className="block px-3 w-full rounded-md border-0 py-1.5 text-gray-900 placeholder:text-gray-400 text-sm sm:leading-6"
+                  style={{ outline: 'none' }}
+                />
+              </div>
+            </div>
+
+            <div>
+              <button
+                type="submit"
+                onClick={handleLogin}
```

Comment:
Since you have added the type submit here you need to add `e.preventDefault` in `handleLogin` function in order to avoid refreshing the page.